Documentation/Docs-CSS/main-section.css

```diff
@@ -0,0 +1,54 @@
+.main {
+  transition: all 0.3s;
+}
+
+.component-section {
+  width: 90vw;
+  padding: 1rem;
+  margin: 1rem auto;
+  height: fit-content;
+  background-color: var(--secondary);
+  color: var(--primary);
+  border-radius: 20px;
+  box-shadow: 6px 6px 12px #131313, -6px -6px 12px #292929;
+}
+
+.blur-filter {
+  filter: blur(20px);
+}
+
+.component-subtitle {
+  margin: 2rem 0;
+}
+
+.component-text {
+  color: var(--white);
+}
+
+.component-display {
+  margin: 2rem 0;
+  padding: 2rem 1rem;
+  background-color: var(--secondary);
+  /* box-shadow: inset 6px 6px 12px #131313, inset -6px -6px 12px #292929; */
```

Comment:
remove code that is unused.

Reply:
> I have suggested some minor changes. Everything else looks good.

Yup! I Will do it. Thanks for the review!